docs/_data/_routes.yml

```diff
@@ -7,136 +7,138 @@ platforms:
   - href: expensify-classic
     title: Expensify Classic
     hub-title: Expensify Classic - Help & Resources
-    url: expensify.com
-    description: Your account settings will look something like this
-    image: /assets/images/paper-airplane.svg
+    url: www.expensify.com
+    description: >- 
+      Your account settings will look something like this:
```

Comment:
This description doesn't seem right. Also I'm not familiar with the YAML syntax you're using here with `>-`. Why not just keep the description on the same line?